code/backend/batch/utilities/helpers/embedders/push_embedder.py

```diff
@@ -38,7 +38,7 @@ def __init__(self, blob_client: AzureBlobStorageClient, env_helper: EnvHelper):
             self.embedding_configs[ext] = processor
 
     def embed_file(self, source_url: str, file_name: str):
-        file_extension = file_name.split(".")[-1]
+        file_extension = file_name.split(".")[-1].lower()
```

Comment:
Can we add/modify a unit test to ensure this is present? Could just uppercase one of the test file extentions https://github.com/Azure-Samples/chat-with-your-data-solution-accelerator/blob/main/code/tests/utilities/helpers/test_push_embedder.py#L129C1-L130C1